File: careerforge-backend/src/main/java/com/careerforge/service/CertificationService.java
package com.careerforge.service;

import com.careerforge.entity.Certification;

import java.util.List;

public interface CertificationService {

    List<Certification> getAll();

    Certification save(Certification certification);

    Certification update(Long id, Certification certification);

    void delete(Long id);

}

File: careerforge-frontend/careerforge-frontend/src/services/certificationService.js
import axiosConfig from "../api/axiosConfig";

const certificationService = {

    getAll: () => axiosConfig.get("/certifications"),

    create: (data) => axiosConfig.post("/certifications", data),

    update: (id, data) => axiosConfig.put(`/certifications/${id}`, data),

    delete: (id) => axiosConfig.delete(`/certifications/${id}`)

};

export default certificationService;